clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5431,7 +5434,7 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
         DisableRangeRounding = true;
     }
     if (DisableRangeRounding || HasFPGA)
-      CmdArgs.push_back("-fsycl-disable-range-rounding");
+      CmdArgs.push_back("-fsycl-range-rounding=disable");
```

Comment:
Given the ability to pass the option via the command line, should we be able to override this setting with what the user wants?

Reply:
I am not sure in the case of FPGA, but maybe using `-fsycl-range-rounding=force` should still generate the range rounded kernel at `-O0`